hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java

```diff
@@ -187,6 +191,8 @@ public Response list(
       response.setTruncated(true);
       ContinueToken nextToken = new ContinueToken(lastKey, prevDir);
       response.setNextToken(nextToken.encodeToString());
+      // For aws api v1 compatibility purpose, set nextMarker with lastKey
```

Comment:
LGTM

// Set nextMarker to be lastKey. for the compatibility  of aws api v1,

Reply:
Done.